pkg/kv/kvserver/split/decider.go

```diff
@@ -23,6 +23,11 @@ const minSplitSuggestionInterval = time.Minute
 const minNoSplitKeyLoggingMetricsInterval = time.Minute
 const minPerSecondSampleDuration = time.Second
 
+type PopularKey struct {
```

Comment:
Could you extend an existing test, or add a new one which asserts on the popular key being returned.

Reply:
Definitely can do 👍